lib/request-handler.spec.js

```diff
@@ -143,25 +152,18 @@ describe('The request handler', function() {
       })
 
       it('should set Cache-Control: no-cache, no-store, must-revalidate if maxAge=0', async function() {
-        process.env.BADGE_MAX_AGE_SECONDS = 0
+        register({ cacheHeaderConfig: { defaultCacheLengthSeconds: 0 } })
         const res = await fetch(`${baseUri}/testing/123.json`)
         expect(res.headers.get('expires')).to.equal(res.headers.get('date'))
         expect(res.headers.get('cache-control')).to.equal(
           'no-cache, no-store, must-revalidate'
         )
       })
 
-      it('should set the expires header to current time + 120 if BADGE_MAX_AGE_SECONDS not set', async function() {
```

Comment:
This functionality was removed. Responsibility for setting the default is now with `server-config`.